tests/ci/benchmark_framework/compare_results.py

```diff
@@ -66,7 +66,7 @@ def main():
     dfs = pd.concat([df1, df2], axis=1)
 
     # Filter out entries with a +15% regression
-    compared = np.where((1 - (df2_avg_time / df1_avg_time)) >= 0.15, df1.iloc[:, 0], np.nan)
+    compared = np.where(((df2_avg_time / df1_avg_time) - 1) >= 0.15, df1.iloc[:, 0], np.nan)
```

Comment:
Should we try `5%` just to see how sensitive it is?

Reply:
I'm fine with dialing it back if we really want to but I suspect that will be noisier than we want. That said, this isn't a required reviewer so we can always adjust it based on the noise generated

Reply:
It would be an experiment